packages/util/src/remove-slashes.spec.ts

```diff
@@ -0,0 +1,17 @@
+import {
+  removeSlashes,
+  removeLeadingSlashes,
+  removeTrailingSlashes
+} from './remove-slashes';
+
+describe('removeSlashes', () => {
+  it('removes trailing slashes', () => {
+    expect(removeSlashes('/test/')).toBe('test');
+  });
+  it('removes leading slashes', () => {
+    expect(removeLeadingSlashes('/test')).toBe('test');
+  });
+  it('removes trailing slashes', () => {
+    expect(removeTrailingSlashes('/test/')).toBe('/test');
+  });
```

Comment:
Just want to comment on these two functions since you touched them. I also used them in my ai-sdk custom destination ticket. I find the naming of these two function however to be very misleading. The functions essentially only remove the **first** or **last** slash, instead of all slashes. The plural form is from my point of view wrong.

Furthermore, all these functions are marked with `@internal` but everything inside util package are exported in `index.ts` and published.

The implementation is 5 years old and `@internal` 3 years old though 😅 @deekshas8

Reply:
But if you change it now, it will break whoever used it. It is a huge breaking change unfortunately... Maybe something for v4.

Reply:
You're right, in general util package functions are exported, so this was probably missed. 
I wasn't aware they are also exposed in `index.ts`, so renaming them would be a breaking change. Something for v4 then.